src/modules/roc_sndio/ibackend.h

```diff
@@ -23,17 +23,21 @@ namespace roc {
 namespace sndio {
 
 //! Backend interface.
+
+/*!
+ * Allows three cases of driver and device combinations.
+ * When driver is NULL and device is NULL, loop through default driver list occurs and we
+ * use "default" device to call open. When driver is NULL and device is not NULL, open is
+ * performed for given driver and output. When driver is not NULL and device is not NULL,
+ * open is performed for given driver and output. A fourth case is not meaningful, hence
+ * it is not supported and is forbidden by command-line.
+ */
 class IBackend {
 public:
     virtual ~IBackend();
 
     //! Probing flags.
     enum FilterFlags {
-        //! Input or output may be a sink.
-        FilterSink = (1 << 0),
-
-        //! Input or output may be a source.
-        FilterSource = (1 << 1),
 
         //! Input or output may be a file.
         FilterFile = (1 << 2),
```

Comment:
Nitpick: since we removed values from beginning, we should adjust the remaining values to start from `(1 << 0)` again.